moses/atomese/representation/atomese_representation.cc

```diff
@@ -0,0 +1,444 @@
+ *
+ * @param contin contin value of a cell
+ * @return
+ */
+Handle mk_real_cell(const contin_t& contin)
+{
+	return createNumberNode(contin)->get_handle();
+}
+
+/**
+ * Represent a real cell as an Execution of the corresponding feature
+ * on its row (i.e Schema of feature on Node of row).
+ *
+ * @param contin contin value of a cell
+ * @param row_num row number
+ * @param label label of the feature
+ * @return
+ */
+Handle mk_real_cell_exec(const contin_t& contin,
+                         const int& row_num, const string& label)
+{
+	return createLink(EXECUTION_LINK,
+	                  createNode(SCHEMA_NODE, label),
+	                  mk_row_number_cell(row_num),
+	                  mk_real_cell(contin));
+}
+
+/**
+ * Create a True / False link based on a given builtin value.
+ *
+ * @param b builtin value
+ * @return
+ */
+Handle mk_boolean_cell(const builtin& b)
+{
+	return createLink(builtin_to_bool(b) ? TRUE_LINK : FALSE_LINK);
+}
+
+/**
+ * Represent a real cell as an Evaluation of the corresponding feature
+ * on its row (i.e Predicate of feature on Node of row).
+ *
+ * @param b builtin value
+ * @param row_num row number
+ * @param label label of the feature
+ * @return
+ */
+Handle mk_boolean_cell_eval(const builtin& b,
+                            const int& row_num, const string& label)
+{
+	Handle h = createLink(EVALUATION_LINK,
+	                      createNode(PREDICATE_NODE, label),
+	                      mk_row_number_cell(row_num));
+	TruthValuePtr tv(SimpleTruthValue::createTV(builtin_to_bool(b), 1));
+	h->setTruthValue(tv);
+	return h;
+}
+
+/**
+ * Represent a boolean cell as an Execution of the corresponding feature
+ * on its row (i.e Schema of feature on Node of row).
+ *
+ * @param b builtin value
+ * @param row_num row number
+ * @param label label of the feature
+ * @return
+ */
+Handle mk_boolean_cell_exec(const builtin& b,
+                            const int& row_num, const string& label)
+{
+	return createLink(EXECUTION_LINK,
+	                  createNode(SCHEMA_NODE, label),
+	                  mk_row_number_cell(row_num),
+	                  mk_boolean_cell(b));
+}
+
+/**
+ * Represent a row as a list of cells.
+ *
+ * @param cells cells in a row
+ * @return
+ */
+Handle mk_row(const HandleSeq& cells)
+{
+	return createLink(cells, LIST_LINK);
+}
+
+/**
+ * Represent a row as a list of row number and cells
+ *
+ * @param cells cells in a row
+ * @param row_num row number
+ * @return
+ */
+Handle mk_numbered_row(const HandleSeq& cells, const int& row_num)
+{
+	return createLink(LIST_LINK,
+	                  mk_row_number_cell(row_num),
+	                  createLink(cells, LIST_LINK));
+}
+
+/**
+ * Create a table of dependent (output) data.
+ *
+ * @param oTable output table
+ * @return
+ */
+Handle mk_output_table(const OTable& oTable)
+{
+	const type_node& type = oTable.get_type();
+	const string& label = oTable.get_label();
+	HandleSeq rows;
+	int row_num = 1;
```

Comment:
I think it's better if it starts by 0 (contrary to what I've indicated on the issue). It makes the subsequent code slightly simpler and it's a more frequent start index value.